### DISCUSSION TITLE
How to graph sparse data points with appropriate intervals?

### DISCUSSION THREAD

**Discussion by Burny-McBurnFace** [tradingview/lightweight-charts#1106] [General]
I have a series of points like:

```
{ time: '2022-01-01', value: '1' },
{ time: '2022-01-02', value: '2' },
{ time: '2022-01-04', value: '3' }
```
Right now I can graph them but the intervals are all the same size. What I want is the x-axis time interval to be 2 times the size for the second points since they are 2 times further. How can I do this?

**Comment by edew** [comment]
@Burny-McBurnFace you can use whitespace data https://tradingview.github.io/lightweight-charts/docs/api/interfaces/WhitespaceData

**Reply by Burny-McBurnFace** [reply]
How do I generate these from the list like the above? The exmple I put in my first post is what I get from the server, is there an API to add whitespace data?
